README.md

```diff
@@ -25,12 +25,15 @@ Running the Back-end & Front-end
     
 Once Docker and Python are Installed then do the following:
     
-    copy server/src/configs/sample_config.py -> server/src/configs/config.py
-    
-Make sure to replace the clientId and clientSecret variables within the config file with
-the ones you were provided, for your application, when you created your Blue Button Sandbox account.
+    cp server/sample-bluebutton-config.json server/.bluebutton-config.json
+
+or (if running docker compose selenium tests)
+
+    cp server/sample-bluebutton-selenium-config.json server/.bluebutton-config.json
```

Comment:
@JFU-GIT I don't think we should mention selenium tests in the main README. This should go under [README-bb2-dev.md](https://github.com/CMSgov/bluebutton-sample-client-python-react/blob/jfuqian/BB2-1702-Python-Sample-App-code-cleanup/README-bb2-dev.md)